expectations.md

```diff
@@ -1 +1,2 @@
 # Expectations
+Маю намір закріпити вже отримані раніше знання та навички, навчитися новому.
```

Comment:
Try to use readme markdown as said @dosandk 
https://dillinger.io/

Reply:
Thank's for recommendations!